scripts/charon_tagged_to_categorical_entities.py

```diff
@@ -0,0 +1,108 @@
+"""
+Script for picking predicted_entities (along with other columns that help with EGA) from charon's tagged_data.csv,
```

Comment:
the description and the script seem totally different, please add the stuff about the categorical entity here.